src/app/client/src/app/modules/shared/services/on-demand-report/on-demand-report.service.ts

```diff
@@ -44,7 +44,7 @@ export class OnDemandReportService {
    */
   getSummeryReports(request: any) {
     const options = {headers: {'Content-Type': 'application/json'}};
-    return this.http.post(this.config.urlConFig.URLS.REPORT_PREFIX + this.config.urlConFig.URLS.REPORT.COLLECTION + this.config.urlConFig.URLS.REPORT.SUMMARY.PREFIX, request, options);
+    return this.http.post(this.config.urlConFig.URLS.REPORT_PREFIX + '/v1' + this.config.urlConFig.URLS.REPORT.COLLECTION + this.config.urlConFig.URLS.REPORT.SUMMARY.PREFIX, request, options);
```

Comment:
Why are using such a long string concatination for a single URL?

Option1: Full URL as on const varaible
Option2:
let constReportURL =  his.config.urlConFig.URLS.REPORT
constReportURL.REPORT_PREFIX + "v1" + REPORT_PREFIX.COLLECTION + constReportUrl.SUMMARY.PREFIX